examples/imagenet/create_imagenet.sh

```diff
@@ -39,6 +48,7 @@ echo "Creating train lmdb..."
 GLOG_logtostderr=1 $TOOLS/convert_imageset \
     --resize_height=$RESIZE_HEIGHT \
     --resize_width=$RESIZE_WIDTH \
+    --encode_type=$TRAIN_ENCODING \
```

Comment:
Check whether this addition would be valuable to the Caffe users. Also check the corresponding additions to the `readme.md` file which improves the tutorial.